### ISSUE TITLE
installer specification problems

### ISSUE THREAD

**Issue by cbaakman** [X-lab-3D/PANDORA#217]
I run the following command to install pandora:
```
conda install -c csb-nijmegen csb-pandora -c salilab -c bioconda
```

I get the following specification errors:

```
UnsatisfiableError: The following specifications were found
to be incompatible with the existing python installation in your environment:

Specifications:

  - csb-pandora -> python[version='>=3.8,<3.9.0a0']

Your python: python=3.9
```
It seems pandora cannot be installed with python 3.9 due to an upper version limit. This would mean that users have to downgrade their python if they want to run pandora. Why can't it be installed for higher versions?

```
The following specifications were found to be incompatible with your system:

  - feature:/linux-64::__glibc==2.31=0
  - feature:|@/linux-64::__glibc==2.31=0

Your installed version is: 2.31
```
I don't understand what's the problem here. My glibc version seems appropriate.

How do I overcome this?

**Comment by Max1461** [comment]
I just made a conda environment with python version 3.8 to get it working.

**Comment by DarioMarzella** [comment]
Hi @cbaakman and @Max1461, I finally found the source of the issue. The python version was properly specified in the package, but I apparently did the conda-build only for python 3.8.

The new pre-release, available now on conda, is compiled for python 3.7-3.10.
